src/Docblock.php

```diff
@@ -184,7 +184,7 @@ protected function parseTags(string $tags): void {
 			foreach (explode("\n", $tags) as $line) {
 				if ($this->isTagLine($line) || count($result) == 0) {
 					$result[] = $line;
-				} else {
+				} elseif ($line !== '') {
```

Comment:
should it be trimmed?

Reply:
Nope, because we lose indentation spaces for multiline comments. I.e. parsing and re-writing a comment like the following

```php

/**
 * @var    Awesome and shiny clear description
 *         of the variable.
 * /
```
results in the(exactly)same docblock.
If we trim, the result will be:

```php

/**
 * @var Awesome  and shiny clear description
 * of the variable.
 * /

Reply:
ah ok... just wanted to be sure, when a check for empty string happens, that usually goes with trim, so we are good 👍